src/wp-includes/fonts/class-wp-font-utils.php

```diff
@@ -29,12 +29,15 @@ class WP_Font_Utils {
 	 * @param string $item A font family name.
 	 * @return string The font family name with surrounding quotes, if necessary.
 	 */
-	private static function maybe_add_quotes( $item ) {
+	public static function normalize_quoted_font_family_name( $item ) {
```

Comment:
I know I suggested this name, but considering again I'd prefer not to include "quoted" here, that seems like an implementation detail of normalization and this function could choose not to use quotes.

This seems preferable, indicating that it's a font family name _for CSS_ which has different encoding rules than other contexts:

```suggestion
	public static function normalize_css_font_family_name( $item ) {
```

---

The description of this function is now incorrect and needs to be updated about its behavior.